pkgs/jni/lib/src/errors.dart

```diff
@@ -100,9 +102,27 @@ final class HelperNotFoundError extends Error {
 
   @override
   String toString() => '''
-Lookup for helper library $path failed.
-Please ensure that `dartjni` shared library is built.
-Provided jni:setup script can be used to build the shared library.
-If the library is already built, ensure that the JVM libraries can be 
-loaded from Dart.''';
+Unable to locate the helper library.
+Ensure that the helper library is available at the path: $path.
+Use the provided jni:setup script to generate the shared library if it does not exist.
+If the library is already built, you may need to explicitly call Jni.spawn with the dylibDir argument set to the directory containing the shared library.
+The directory containing the shared library is typically in build\\jni_libs.
```

Comment:
nit: Since the error message is for all operating systems, this can be `build${Platform.pathSeparator}jni_libs` instead.

Reply:
smart